lib/elixir/lib/system.ex

```diff
@@ -465,10 +465,12 @@ defmodule System do
   `command` is expected to be an executable available in PATH
   unless an absolute path is given.
 
-  `args` must be a list of strings which are not expanded
-  in any way. For example, this means wildcard expansion will
-  not happen unless `Path.wildcard/2` is used. On Windows though,
-  wildcard expansion is up to the program.
+  `args` must be a list of binaries which the executable will receive
+  as its arguments as-is. This means that
+
+    * Environment variables will not be interpolated.
+    * Wildcard expansion will not happen (unless `Path.wildcard/2` is used explicitly).
+    * Arguments do not need to be escaped or quoted for shell safety.
 
```

Comment:
Sorry, I meant to mention this in the commit note.

I omitted it as it seemed odd that we mention it’s up to the program under Windows, as it’s the case everywhere (we even mention using Path.wildcard/2 to trigger it). Maybe there’s something I misunderstood about the original note?